jami-android/app/src/main/java/cx/ring/service/AccountData.kt

```diff
@@ -0,0 +1,86 @@
+package cx.ring.service
+
+class AccountData {
```

Comment:
I'd put this code on our side of the service, and feed the map through aidl I'd say, that way it's easier to change.

Reply:
Hi, 
I spend almost all day making map forwarding work, it is not such trivial as I expected. How much this is important to be done before the release?

Reply:
I just realized I do not run Jami account creation from our Setup flow, need to think where better to call it.